src/Class.js

```diff
@@ -1,10 +1,10 @@
 var WebJack = {};
 
-(function(exports){
```

Comment:
It's not for server compatibility, just in case ppl want to include it in
another node package that they write, using the AMD standard. Cool, thanks!
On Jun 6, 2016 5:58 PM, "Richard Meister" <notifications@github.com> wrote:

> In src/Class.js
> <https://github.com/publiclab/webjack/pull/1#discussion_r65977196>:
>
> > @@ -1,10 +1,10 @@
> >  var WebJack = {};
> >
> > -(function(exports){
>
> Changed it like in the example, now it does not throw an error.
>
> I don't know if there is a better way, but I wonder why we need server
> compatibility?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/webjack/pull/1/files/2825bdb6cb367b242931faf8641cf2647704ddfa#r65977196>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABfJ1gIEUwLbtcUsyKSsv0EOigrkQAcks5qJJfsgaJpZM4ItzEp>
> .
>